convert2rhel/unit_tests/actions/post_conversion/rhsm_custom_facts_config_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import pytest
+import six
+
+from convert2rhel import actions, subscription, utils
+from convert2rhel.actions.post_conversion.rhsm_custom_facts_config import RHSMCustomFactsConfig
+
+
+# from convert2rhel.unit_tests import RunSubprocessMocked
+
+
```

Comment:
```suggestion
```

Reply:
gotcha, i will make the appropriate change.